### PR TITLE
TOK-517: pause builder reward claim

### PR DESCRIPTION
**Pull request by franciscotobar**
[TOK-517](https://rsklabs.atlassian.net/browse/TOK-517)

[Depends on](https://github.com/RootstockCollective/dao-frontend/pull/430)

### CODE REVIEW

File: src/app/collective-rewards/rewards/builders/hooks/useClaimBuilderRewards.ts [hunk 1]
```diff
@@ -3,9 +3,15 @@ import { GaugeAbi } from '@/lib/abis/v2/GaugeAbi'
...
 import { useGetBuilderRewards } from '@/app/collective-rewards/rewards'
+import { useBuilderContext } from '../../../user'
 
-const useClaimBuilderReward = (gauge: Address, rewardToken?: Address) => {
+const useClaimBuilderReward = (builder: Address, gauge: Address, rewardToken?: Address) => {
   const { writeContractAsync, error: executionError, data: hash, isPending } = useWriteContract()
+  const { getBuilderByAddress } = useBuilderContext()
+
+  const claimingBuilder = getBuilderByAddress(builder)
+  console.log('🚀 ~ useClaimBuilderReward ~ claimingBuilder:', claimingBuilder)
```
Comment (jurajpiar) [inline src/app/collective-rewards/rewards/builders/hooks/useClaimBuilderRewards.ts:13]:
```suggestion
```

File: src/app/collective-rewards/rewards/builders/hooks/useClaimBuilderRewards.ts [hunk 2]
```diff
@@ -3,9 +3,15 @@ import { GaugeAbi } from '@/lib/abis/v2/GaugeAbi'
...
+  const isPaused = !!claimingBuilder?.stateFlags?.paused
```
Comment (jurajpiar) [inline src/app/collective-rewards/rewards/builders/hooks/useClaimBuilderRewards.ts:14]: is it ok to show v1 (no flags) as paused?

**Review by jurajpiar — APPROVED** [review]
LG

File: src/app/collective-rewards/rewards/builders/hooks/useClaimBuilderRewards.ts (hunk 2 above)
Comment (franciscotobar) [reply src/app/collective-rewards/rewards/builders/hooks/useClaimBuilderRewards.ts:14]: not really, let me update that, but at this point I dont think that they could see this.

**Pull request merged**